doc/source/turn-off.rst

```diff
@@ -7,18 +7,34 @@ When you are done with your hub, you should delete it so you are no longer
 paying money for it. The following sections describe how to delete your
 JupyterHub resources on various cloud providers.
 
-Google Cloud Engine
--------------------
+For all cloud providers
+-----------------------
+
+The steps in this section must be performed for all cloud providers first,
+before doing the cloud provider specific setup.
+
+1. First, delete the helm release. This deletes all resources that were created
+   by helm to make your jupyterhub.
+
+  .. code-block:: bash
+
+     helm delete <your-helm-release-name> --purge
 
-1. First, delete the namespace the hub was installed in. This deletes any disks
+2. First, delete the namespace the hub was installed in. This deletes any disks
    that may have been created to store user's data, and any IP addresses that
    may have been provisioned.
 
    .. code-block:: bash
 
       kubectl delete namespace <your-namespace>
 
-2. Next, you should delete the kubernetes cluster. You can list all the clusters
+Google Cloud Engine
+-------------------
+
+Make sure you have performed the cloud provider agnostic steps listed above before
```

Comment:
Make this line an actual step. Maybe:

1. Perform the steps above for all cloud providers. These cloud provider agnostic steps will delete the helm chart and delete the hub's namespace. This must be done before proceeding.